corehq/tests/test_elastic.py

```diff
@@ -43,6 +41,40 @@ def test_scroll_query(self):
                 self.assertEqual(result["_source"], indexed[result["_id"]])
             self.assertEqual(len(results), len(indexed))
 
+    def test_scroll_query_returns_over_2x_size_docs(self):
+        """Test that all results are returned in very large scroll queries.
+
+        NOTE: this test is slow (~7sec on an i7-10610U)
```

Comment:
Would it be possible to temporarily reduce the scroll limit from 2000 to something much lower (like 5) just for this test so you don't have to create so many records to reproduce the issue?